docs/getting-started/benchmark.md

```diff
@@ -0,0 +1,214 @@
+We used the same `c7gn.16xlarge` for running `memtier_benchmark` and we used the same commands
+to test writes, reads and pipelined reads:
+
+| Test          | Ops/sec   | Avg. Latency (us) | P99.9 Latency (us) |
+|---------------|-----------|-------------------|--------------------|
+| Write-Only    | 5.2M      | 250               | 631                |
+| Read-Only     | 6M        | 271               | 623                |
+| Pipelined Read| 8.9M      | 323               | 839                |
+
+
+### Writes
+`memtier_benchmark -s $SERVER_PRIVATE_IP --distinct-client-seed --hide-histogram --ratio 1:0 -t 60 -c 20 -n 200000`
+
+```
+============================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec
+----------------------------------------------------------------------------------------------------------------------------
+Sets      5195097.56          ---          ---         0.26012         0.24700         0.49500         0.63100    400230.15
+```
+
+### Reads
+`memtier_benchmark -s $SERVER_PRIVATE_IP --distinct-client-seed --hide-histogram --ratio 0:1 -t 60 -c 20 -n 200000`
+```
+============================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec
+----------------------------------------------------------------------------------------------------------------------------
+Gets      6078632.89   6078632.89         0.00         0.27177         0.26300         0.49500         0.62300    438616.86
+```
+
+### Pipelined Reads
+`memtier_benchmark -s $SERVER_PRIVATE_IP --ratio 0:1 -t 60 -c 5  -n 200000  --distinct-client-seed --hide-histogram --pipeline=10`
+
+```
+============================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec
+----------------------------------------------------------------------------------------------------------------------------
+Gets      8975121.86   8975121.86         0.00         0.32325         0.31100         0.52700         0.83900    647619.14
+```
+
+
+## Garnet on `c6in.12xlarge`
+[Garnet](https://github.com/microsoft/garnet) has been recently released by Microsoft Research
+and people are curious how it compares with Dragonfly. We are not going to cover here about
+the architectural differrences between Garnet and Dragonfly, and their implications on compatibility
+with Redis but we provide a brief summary of Garnet performance and compare it with Dragonfly.
+
+Unfortunately, Garnet does not have aarch64 build available so we run it on intel-powered server
```

Comment:
Consider doing the whole page using x64, as otherwise it's just incomparable and users will either be confused or angry that they are "being mislead" (even though that is clearly not the goal here)

Reply:
I added Dragonfly results below for the same I stance, I have not finished.